apps/backend/app.py

```diff
@@ -185,12 +201,37 @@ def upload_experiment_results(expId, trialExtraFile, postProcess):
     uploadBlob = firebaseBucket.blob(f"results/result{expId}.csv")
     uploadBlob.upload_from_filename('results.csv')
 
+        # Upload to MongoDB
+
+    try:
+            mongoClient.admin.command('ping')
+    except ConnectionFailure:
+        print("Server not available (this is an error)")
```

Comment:
re-raise as GladosInternalError (or a new exception type you make) in addition to printing